core/src/test/java/org/apache/iceberg/TestTableMetadata.java

```diff
@@ -402,7 +396,7 @@ public void testInvalidMainBranch() throws IOException {
             () ->
                 new TableMetadata(
                     null,
-                    MAX_FORMAT_VERSION,
```

Comment:
I think these need to be kept as MAX_FORMAT_VERSION

Reply:
I kept MAX_FORMAT_VERSION where the row ID value was non-zero, but I may have gone too far for some cases where it should be the latest format with zero data files.

Reply:
Yeah, I see. These are testing cases that throw errors. I'll fix them.

Reply:
Fixed.